ttl/ttl.go

```diff
@@ -87,7 +87,7 @@ func (tm *TTL) ttl(ctx context.Context, key any) {
 }
 
 // Put inserts the provided key and value into the TTL map. The ttl values
-// designates the duration of the validity of this key value pair. The expired
+// designates the duration of the validity of this key value-pair. The expired
```

Comment:
I don't believe "value-pair" is correct, however key value should be written as "key-value":
https://en.wikipedia.org/wiki/Name%E2%80%93value_pair
```suggestion
// designates the duration of the validity of this key-value pair. The expired
```